uber/models.py

```diff
@@ -1338,6 +1342,15 @@ def requested(self, department):
     def assigned_to(self, department):
         return int(department or 0) in self.assigned_depts_ints
 
+    def trusted_in(self, department):
+        return int(department or 0) in self.trusted_in_ints
+
```

Comment:
I would be inclined to make "assigned_to`` part of the ``trusted_in`` check.  Like, is there any situation where someone would be trusted in a department but NOT assigned there?  (See my note above about automatically unsetting someone as trusted from a department when they're not assigned there.)